backend/events/matsucon2024/forms/vendor-application-en.yml

```diff
@@ -97,6 +97,11 @@ fields:
     type: FileUpload
     required: true
 
+  - slug: additional_info
```

Comment:
add helpText: "For any comments or info that didn't fit the categories above"

Reply:
added